CITATION.cff

```diff
@@ -596,6 +596,8 @@ authors:
     given-names: Lorenzo
   - family-names: Hejtmánek
     given-names: Lukáš
+  - family-names: Jack)
+    given-names: Lumberbot (aka
```

Comment:
this looks wrong

Reply:
Yeah I saw that, not sure it's matters because it's a bot. I guess we could special-case it in the parser, what do you want `Lumberbot (aka Jack)` to have for `family-names` and `given-names`? Or maybe we should just exclude it?

Reply:
Never mind I see we already exclude `[bot]` so I'll just make it exclude!